### PR TITLE
Configure linters

### LINKED ISSUE #24: add linting infra
Using `eslint-config-bliss` and `eslint-config-airbnb`

### PR DESCRIPTION
**Pull request by vikr01**
Closes #24

### CODE REVIEW

File: website/siteConfig.js [hunk 1]
```diff
@@ -2,40 +2,45 @@ const users = [];
...
+    secondaryColor: '#030C1C',
   },
 
   // This copyright info is used in /core/Footer.js and blog RSS/Atom feeds.
   copyright: `Copyright © ${new Date().getFullYear()} Amila Welihinda`,
 
   highlight: {
     // Highlight.js theme to use for syntax highlighting in code blocks.
-    theme: 'tomorrow'
+    theme: "tomorrow"
```
Comment (amilajack) [inline website/siteConfig.js]: Double quotes are used here but single quotes are used in other places.

File: .eslintrc.json [hunk 2]
```diff
@@ -0,0 +1,23 @@
+{
+  "extends": ["airbnb", "bliss", "prettier"],
+  "rules": {
+    "class-methods-use-this": "warn",
+
+    "import/no-dynamic-require": 0,
```
Comment (amilajack) [inline .eslintrc.json]: Using the error/warn/info enums is better for eslint configs

Comment (vikr01) [reply .eslintrc.json]: Do you mean error/warn/**off**?

Comment (amilajack) [reply .eslintrc.json]: oh yea that

**Comment by amilajack** [comment]
🎉 🎉 🎉  Good stuff!

**Pull request merged**